pontoon/insights/tasks.py

```diff
@@ -120,9 +120,9 @@ def get_contributors():
 
     Note that excluding system user emails in the Translation QuerySet directly is slow.
     """
-    system_users = User.objects.filter(
-        email__regex=r"^pontoon-(\w+)@example.com$",
-    ).values("pk")
+    system_users = UserProfile.objects.filter(system_user=True).values_list(
+        "user_id", flat=True
+    )
```

Comment:
You could also leave the original structure and just replace the second line with:
```python
profile__system_user=True
```